AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/main/Footer.kt

```diff
@@ -0,0 +1,29 @@
+package com.applovin.enterprise.apps.demoapp.data.main
+
+import android.os.Build
+import android.os.Build.VERSION_CODES
+import com.applovin.enterprise.apps.demoapp.BuildConfig
+import com.applovin.sdk.AppLovinSdk
+
+
+/**
+ * A [ListItem] representing a footer on the main screen
+ * <p>
+ * Created by Jason Zheng on 7/15/2020.
+ */
+data class Footer(override var type: Int = ListItem.FOOTER) : ListItem
```

Comment:
Looks like we are overriding `type` differently in other `ListItem` implementations. Do you mind updating the others to this or vice versa. Should always keep the style consistent.

```suggestion
data class Footer(override val type: Int = ListItem.FOOTER) : ListItem
```